test/property.test.js

```diff
@@ -62,37 +58,127 @@ describe('loopback:property generator', function() {
     });
   });
 
-  it('creates a typed array', function(done) {
+  it('should create model containing number type', function(done) {
     var propertyGenerator = givenPropertyGenerator();
     helpers.mockPrompt(propertyGenerator, {
       model: 'Car',
-      name: 'list',
+      name: 'age',
+      type: 'number',
+      required: 'true',
+      defaultValue: '555555555555555555555.5'
```

Comment:
I see. In that case, please make the test name explicitly mention that you are testing a very large default value. E.g. `should create model containing number type with very large default value`